Tests and docs

rmarkello opened the pull request:
Created tests for `utils.py` (see: `test_utils.py`) and modified some docstrings in the relevant file. No code was changed other than to bring it to PEP8 compliance.

rmarkello committed:
[TEST]: Added tests (and some docs)

Created tests for `utils.py` and modified some doc-strings in the relevant file

emdupre commented:
This is great, thank you ! Could you please also change the `.travis.yml` to find your tests? It should be:
```
py.test --ignore=tests/test_tedana.py
```

rmarkello committed:
[TEST]: Update .travis.yml

Modified .travis.yml to ignore `test_tedana.py` while incorporating new test
files.
